dev/tools/svg2dart/bin/main.dart

```diff
@@ -0,0 +1,89 @@
+import 'dart:io';
+
+import 'package:args/args.dart';
+import 'package:svg2dart/svg2dart.dart';
+
+const String kCodegenComment =
+  '// AUTO GENERATED FILE DO NOT EDIT!\n'
+  '// This file was generated by svg2dart.\n';
+
+void main(List<String> args) {
+  final ArgParser parser = new ArgParser();
+
+  parser.addFlag(
+      'help',
+      abbr: 'h',
+      negatable: false,
+      help: 'Display the tool\'s usage instructions and quit.');
+
+  parser.addOption(
+      'output',
+      abbr: 'o',
+      help: 'Target path to write the generated Dart file to.');
+
+  parser.addOption(
+      'asset_name',
+      abbr: 'n',
+      help: 'Name to be used for the the generated const.');
+
+  parser.addOption(
+      'part_of',
+      abbr: 'p',
+      help: 'Library name to add a dart \'part of\' clause for');
+
+  parser.addOption(
+      'header',
+      abbr: 'd',
+      help: 'The contents of this file will be prepended to the beginning of '
+            'the generated Dart file, this can be used for a license comment'
```

Comment:
the comma here should be a period or semicolon

Reply:
done